db/seeds.rb

```diff
@@ -0,0 +1,28 @@
+require 'faker'
```

Comment:
You can require  faker on config/environment, the place where we should require all the gems we're going to use!